test/test_files/tinysnb/function/string.test

```diff
@@ -687,3 +687,51 @@ Deswork
 Sóló cón tu párejâ
 The 😂😃🧘🏻‍♂️🌍🌦️🍞🚗 movie
 Roma
+
+-LOG StringSplitFunc
+-STATEMENT match (p:person) return string_split(p.fName, 'a');
+---- 8
+[Alice]
+[Bob]
+[C,rol]
+[D,n]
+[Eliz,beth]
+[F,rooq]
+[Greg]
+[Hubert Bl,ine Wolfeschlegelsteinh,usenbergerdorff]
+
+-LOG StringSplitFunc
+-STATEMENT match (p:person) return str_split(p.fName, 'li');
```

Comment:
Since you implement str_split as binary function. Can u try a str_split(<non_literal>, <non_literal>)